app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def inner(*args) -> int:
```

Comment:
Apply a decorator from `functools` to the `inner` to preserve the wrapped function name and docstring.